### DISCUSSION TITLE
Welcome to retrofw.github.io Discussions!

### DISCUSSION THREAD

**Discussion by pingflood** [retrofw/retrofw.github.io#44] [Dev talk]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Poligraf** [comment]
@pingflood wanted to check what your plan for future is for retrofw.
1) blackz is loking at the dev version but we havent made progress on finding why the DEV version clocks down to 528 when headphones plugged in. probably something to do with tv out.any help appreciated.

2)also there is a pull request waiting from glebm for a couple of months now. the sdl ttf and image packages I was getting a 404 because they are pointing to a dead link
